### DISCUSSION TITLE
Completely broken

### DISCUSSION THREAD

**Discussion by Bandit253** [fastapi/fastapi#7380] [Questions]
Sorry for the terrible title but I couldn't think of a better one :( 

I had developed a site and it was working really well but something happened and it now no longer works.  In order to identify the issue I have started again with absolute minimum 
`from fastapi import FastAPI
import uvicorn

app = FastAPI()
@app.get("/")
async def read_root():
    return {"Hello": "World"}

if __name__ == "__main__":
    uvicorn.run(app, host="127.0.0.1", port=8000)`

Python 3.8.4 -- Is this the problem? Something I did asked me to go from .3 to .4 ???
uvicorn --version
Running uvicorn 0.11.5 with CPython 3.8.3 on Windows

It does not work or return anything, the arrow just spins.
In my more complex site, it not only does not work it folds up into a 'none existent' default endpoint.

Try this site:
https://rdshwys.azurewebsites.net/docs# to demonstrate the issue

![image](https://user-images.githubusercontent.com/9003261/88348330-7f1fda00-cd90-11ea-975e-64417e3bcf54.png)

**Comment by Bandit253** [comment]
@niko1a I understand your confusion, thus the title!

But there is an update!  I changed the python version, way back to 3.6.9 (it was one I have available) and it worked fine.

From this I think it is due to Python 3.8.4

**Comment by Bandit253** [comment]
Ok I have solved my issue.

I just uninstalled pytest and it works again.  Pytest was the thing that asked for the upgrade from 3.8.3 to 3.8.4

**Comment by Bandit253** [comment]
It shouldn't have broken it but Pytest was the problem.

I even downgraded back to 3.8.3 and still didn't work.

**Comment by phy25** [comment]
Duplicate of #1770.

**Comment by tiangolo** [comment]
Thanks for the help here @phy25 ! :clap: :bow: 

Thanks for reporting back and closing the issue :+1:
